frontend/ui/screensaver.lua

```diff
@@ -146,46 +150,68 @@ function Screensaver:show()
     end
     if screensaver_type == "cover" then
         local lastfile = G_reader_settings:readSetting("lastfile")
-        local doc_settings = DocSettings:open(lastfile)
-        local exclude = doc_settings:readSetting("exclude_screensaver")
+        local exclude = false -- consider it not excluded if there's no docsetting
+        local remove_sidecarfile = false
+        if DocSettings:hasSidecarFile(lastfile) then
+            local doc_settings = DocSettings:open(lastfile)
+            exclude = doc_settings:readSetting("exclude_screensaver")
+            doc_settings:close()
+        else
+            remove_sidecarfile = true
```

Comment:
Not sure we need this `else remove_sidecarfile=true` here and below (`if DocSettings:hasSidecarFile(lastfile) ... else -- there is no sidecar file to remove :)`)
Or did I miss something ?